ms_office_macro_bleach/bleach.py

```diff
@@ -0,0 +1,76 @@
+"""This module removes any and all macros/dynamic content from MS Office files.
+
+It is part of a suite of programs developed by the AntiMalware Alliance.
+
+Visit https://github.com/Anti-Malware-Alliance for more details about our organisation and projects.
+
+Feel free to contact benjamin.mcgregor2002@gmail.com for any questions regarding this module.
+"""
+from argparse import ArgumentParser
+from os import rename, path, remove
+from zipfile import ZipFile
+from shutil import make_archive, rmtree
+
+
+# Unzip file function
+# Converts file to a .zip archive and extracts it to file directory
+def unzip_file(file):
+
+    # Convert to zip archive
+    rename(file, file + ".zip")
+
+    # Extract contents into temporary folder
+    with ZipFile(file + ".zip", 'r') as zip_ref:
+        zip_ref.extractall(file + "_temp")
+
+    # Delete original file
+    remove(file + ".zip")
+
+
+# Remove macros function
+# Checks for macro files and deletes them if found
+def remove_macros(file, notify):
+    macros_found = False
+
+    # Deletes macro binary file if found
+    if path.exists(file + "_temp/word/vbaProject.bin"):
+        remove(file + "_temp/word/vbaProject.bin")
```

Comment:
Do you know if Macros are always called this? Or if this name can be manipulated to bypass. I remember that all Macros were placed on the same container.

Reply:
I've tested with multiple macros at a time, so far all their data and definitions is stored in these two files. By default, word stores created macros in a global location, but if you import them into the document itself, its always the same location.

Reply:
Good to know. I will also do a bit of my own testing in the future, to see if we need to acomodate for more cases.